src/main/java/com/scalar/db/storage/cosmos/CosmosTableMetadata.java

```diff
@@ -1,34 +1,24 @@
 package com.scalar.db.storage.cosmos;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSortedSet;
-import com.scalar.db.api.Scan;
-import com.scalar.db.exception.storage.UnsupportedTypeException;
-import com.scalar.db.storage.common.metadata.DataType;
-import com.scalar.db.storage.common.metadata.TableMetadata;
 import com.scalar.db.storage.common.util.ImmutableLinkedHashSet;
-
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.stream.Collectors;
 
 /**
  * A metadata class for a table of Scalar DB to know the type of each column
  *
  * @author Yuji Ito
  */
-public class CosmosTableMetadata implements TableMetadata {
+public class CosmosTableMetadata {
```

Comment:
Just a question.  Why this cannot be unified to `TableMetadata`?

Reply:
Thank you for reviewing this!

We still need `CosmosTableMetadata` because we need POJO (that means a class with some fields and their getters and setters maybe) to serialize and deserialize data in Cosmos DB.

For example, the following code writes the metadata into Cosmos DB with `CosmosTableMetadata`:
https://github.com/scalar-labs/scalardb/blob/2ff0ff36163158fd17627d476fcca7bd9c4bbba8/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosIntegrationTest.java#L85-L100

And the following code retrieves the metadata from Cosmos DB with `CosmosTableMetadata`:
https://github.com/scalar-labs/scalardb/blob/2ff0ff36163158fd17627d476fcca7bd9c4bbba8/src/main/java/com/scalar/db/storage/cosmos/CosmosTableMetadataManager.java#L49-L51

As the new `TableMetadata` is not POJO, we can't use it for that.

Reply:
Thank you! I see.
Since `CosmosTableMetadata` is only used in `CosmosTableMetadataManager`, can it be an inner class of the manager class?

Reply:
`CosmosTableMetadata` is used only in `CosmosTableMetadataManager ` in the main code, but it's actually used in the integration test code as well. And we might use this class another part when we introduce `DistributedStorageAdmin`. So I want to keep it an outer class for now. If we realize it doesn't need to be an outer class in the future, we can do that at that time. Thanks.

Reply:
OK, I understand.